znoyder/config.yml

```diff
@@ -808,8 +810,6 @@ override:
       'osp-rpm-py39':
         voting: true
         vars:
-          rhos_release_args: '17.0'
-          rhos_release_extra_repos: rhelosp-17.0-trunk-brew
```

Comment:
Have we tested the tripleo-ansible without this variables? Or are we sure that this extra repos are not needed with rpm jobs?